components/main/Main.tsx

```diff
@@ -82,6 +83,7 @@ export const Main = ({ signOut, appVersion }: IMain) => {
             <Route exact path="/home" component={Home} />
             <Route exact path="/placements" component={Placements} />
             <Route exact path="/programmes" component={Programmes} />
+            <Route exact path="/programmes/:id" component={CojView} />
```

Comment:
The route should indicate that it is for signing coj e.g. `/programme/:id/sign-coj`